src/components/Referrals/ReferralHeroSection/ReferralHeroSection.spec.tsx

```diff
@@ -29,8 +30,35 @@ jest.mock('../../../modules/routing/locations', () => ({
 }))
 
 describe('ReferralHeroSection', () => {
-  const profileAddress = '0x123456789abcdef'
-  const inviteUrl = `${INVITE_REFERRER_URL}/${profileAddress}`
+  let profileAddress: string
+  let profileAddressWithClaimedName: string
+  let inviteUrl: string
+  let mockAvatarWithClaimedName: Avatar
+  let mockAvatarWithoutClaimedName: Avatar
+
+  beforeEach(() => {
+    profileAddress = '0x123456789abcdef'
+    profileAddressWithClaimedName = '0x123456789abcdef'
```

Comment:
`profileAddressWithClaimedName` is the same as `profileAddress`, what do you think about removing it?